assemblyscript/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "wasi-nn",
```

Comment:
@brianjjones, just a thought: should we rename the NPM version of this to `as-wasi-nn`? That would 1) match the `as-wasi` naming, and 2) make it more clear that this is an AssemblyScript package, not a JS or TS one.

Reply:
I have no issue with that. Should the rust version be rust-wasi-nn then?

Reply:
Everything on crates.io is pretty much Rust so I don't think so.